roles/sap_ha_pacemaker_cluster/tasks/construct_vars_hana_common.yml

```diff
@@ -24,7 +24,7 @@
   vars:
     __resource_hana_topology:
       id: "{{ sap_ha_pacemaker_cluster_hana_topology_resource_name }}"
-      agent: "ocf:heartbeat:SAPHanaTopology"
+      agent: "ocf:{{ __sap_ha_resource_agent_namespace }}:SAPHanaTopology"
```

Comment:
May warrant same approach (using dictionary lookup from the OS Family name) as all other variable handling, for consistency? See further comments below.

Reply:
@sean-freeman  Yes, but cluttering it with whole 2 lines of IF conditional will make it less readable, using variable (comment can be added) is clearer.

Reply:
I agree this needs adjustment to move hard-coded paths and agent names from the code into vars files. 
However, I would do this later, as it requires additional review and planning of scenarios covered and how we define the specific details most efficiently without rewriting it again fairly soon (e.g. future combined agent for scale-up and scale-out, currently in upstream development).

Reply:
OK, leaving for now